lib/src/facebook_login.dart

```diff
@@ -157,6 +158,14 @@ class FacebookLogin {
     return FacebookLoginResult.fromMap(loginResultData.cast<String, dynamic>());
   }
 
+  Future<FacebookLoginResult> expressLogin() async {
```

Comment:
We need to add documentation with mentions that this is only available for Android.